test/index.test.js

```diff
@@ -54,7 +54,6 @@ describe('Uploader Core', () => {
   it('Can parse environment variables', async () => {
     process.env.SOMETHING = 'red'
     process.env.ANOTHER = 'blue'
-    jest.spyOn(process, 'exit').mockImplementation(() => {})
     const log = jest.spyOn(console, 'log').mockImplementation(() => {})
     await app.main({
```

Comment:
`test/index.test.js:58:5`

https://app.circleci.com/pipelines/github/codecov/uploader/678/workflows/96eeefce-0f14-4972-988e-2b3dd3039c7f/jobs/4549?invite=true#step-105-43

Ok, so it _has_ to be mocked. 👎

Reply:
```suggestion
     const log = jest.spyOn(process, 'exit').mockImplementation(() => {})
    await app.main({
```

Reply:
Going to try mocking just this one, to see if the error moves.